src/app/mock/logs.ts

```diff
@@ -1,3 +1,3 @@
-import { Log } from '../models/log';
+import { Log } from '../models/Log';
```

Comment:
To be completely correct, this import statement is not used in this file and can be completely removed, as the `LogData` object is untyped.

But the better fix would be to have a field on `Log`, something like `message: string` so we have a minimal structure for the log message.
But this is a good catch, thanks, @bacongobbler !

Reply:
okay, I'll go ahead and remove this for now and circle back once I have a better idea on the architecture :)